actions/kustomize-validation/kustomize-build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+# Run kustomize build for all folders containing a kustomization.yaml file
+while read -r -d '' file; do
+    dir=$(dirname "$file")
+    echo -e "\n======================================================================================" 1>&2
+    echo "Running  'kustomize build '$dir':" 1>&2
+    echo "======================================================================================" 1>&2
+    ./kustomize build "$dir"
```

Comment:
Take a look here: https://fluxcd.io/docs/faq/#what-is-the-behavior-of-kustomize-used-by-flux

We should add the flags `--load-restrictor=LoadRestrictionsNone --reorder=legacy` to adhere with what Flux uses.

Reply:
I added a "options" parameter to the action to make that possible